Core.Collectors/Web/AdlsClientWrapper2.cs

```diff
@@ -6,7 +6,12 @@
 
 namespace Microsoft.CloudMine.Core.Collectors.Web
 {
-    public class AdlsClientWrapper2 : IAdlsClient
+    public interface IAdlsClient2
+    {
+        public AdlsClient AdlsClient { get; }
```

Comment:
Where is it used?

Reply:
It will be used for dual/multi ingestion. Could not figure out a better way to use it via dependency injection. Happy to discuss a re-design.